yb-voyager/src/srcdb/postgres.go

```diff
@@ -463,6 +463,16 @@ func (pg *PostgreSQL) GetCharset() (string, error) {
 	return encoding, nil
 }
 
+func (pg *PostgreSQL) GetDatabaseSize() (int64, error) {
+	var dbSize int64
+	query := fmt.Sprintf("select pg_database_size('%s'); ", pg.source.DBName)
```

Comment:
I have read about all three of them, and most are generally fast as they all read the database's metadata to give size but the Oracle one where I am accessing `DBA_SEGMENTS` is summing up all the segments in the schema so if those are large in number it might become a bit slow. But I don't think there can be any other option for this in Oracle.

Reply:
Okay. Either way this would be a one-time operation at the beginning of export-schema right?